### PR TITLE
Add suffixes to typos from "ea" to "en"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -21170,7 +21300,12 @@ enhacing->enhancing
 enhancd->enhanced
 enhancment->enhancement
 enhancments->enhancements
+enhane->enhance
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21303]: Perhaps `ethane`, although **N** and **T** are not neighbours on my keyboard?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -21183,6 +21318,10 @@ enities->entities
 enitities->entities
 enitity->entity
 enitre->entire
+enitrely->entirely
+enitrety->entirety
+enitties->entities
+enitty->entity
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21324]: Perhaps `snitty`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -21438,8 +21611,11 @@ envrironments->environments
 envrirons->environs
 envvironment->environment
 enxt->next
+enymore->anymore
 enything->anything
+enytime->anytime
 enyway->anyway
+enywhere->anywhere
 epecifica->especifica
 epect->expect
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21620]: Perhaps add `elect` and even `eject` and `erect` as fixes for `epect`?

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:21324]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:21620]: I found `elect` but not `eject` or `erect`. I'll add `elect` to #3049.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**